src/pages/terms.mdx

```diff
@@ -26,551 +18,259 @@ PLEASE READ THIS AGREEMENT CAREFULLY BEFORE PURCHASING AND/OR USING SOFTWARE OR
+
+## 1. Definitions
+
+1.1. **Problem** means a failure of the Supported Software to substantially conform to the functional specifications set forth in the applicable documentation provided by ReductStore.
+
+1.2. **Support Services** means each of the services made available by us under this Agreement including those offered via Architecture Review, software support and problem resolution as part of the Service offerings.
 
-## <a id="addendum-1-definitions"></a>1. Definitions
+1.3. **Updates and Upgrades** means subsequently released versions of the Supported Software. For clarity, Updates and Upgrades do not include any products that ReductStore licenses separately from the Supported Software or for an additional fee.
 
-1.1. **Problem** means a failure of the Supported Software to substantially conform to the functional specifications
-set forth in the applicable documentation provided by ReductStore.
+1.4. **Workaround** means a modification or "patch" for a particular version of the Supported Software, which may be of a temporary or interim nature, to help avoid, correct or circumvent a Problem.
 
-1.2. **Support Services** means each of the services made available by us under this Agreement including those offered
-via Annual Solution Review, software support and problem resolution as part of the Service offerings.
+## 2. Support Services
 
-1.3. **Updates and Upgrades** means subsequently released versions of the Supported Software. For clarity, Updates and
-Upgrades do not include any products that ReductStore licenses separately from the Supported Software or for an
-additional fee.
+2.1. **Eligibility**. Upon payment of the applicable Fees, ReductStore shall provide Support Services as further described herein. To be eligible to receive Support Services in accordance with this Addendum.
 
-1.4. **Workaround** means a modification or "patch" for a particular version of the Supported Software, which may be of
-a temporary or interim nature, to help avoid, correct or circumvent a Problem.
+2.2. **Maintenance**; Updates and Upgrades. As part of the Support Services, ReductStore shall use commercially reasonable efforts to maintain the Supported Software so that it operates without Problems. ReductStore shall supply Customer with Guided Updates and Upgrades for the Supported Software during the Subscription Term. Such Updates and Upgrades shall be provided to Customer at no charge.
 
-## <a id="addendum-2-support-services"></a>2. Support Services
+2.3. **Support**. Subject to Section 2.1 of this Addendum, ReductStore shall provide the Support Services described herein to assist in Customer’s use of the Supported Software and resolve Problems in the Supported Software through direct engineering support delivered remotely via the Internet. ReductStore agrees to use commercially reasonable efforts to acknowledge the Problems reported to ReductStore by Customer:
```

Comment:
We can use **Problem Resolution** definition here to simplify the text. For example: to assist the Customer in Problem Resolution through direct engineering support delivered remotely via the Internet.